lib/service_sign_in/check-income-tax.en.yaml

```diff
@@ -10,6 +10,9 @@ choose_sign_in:
     - text: Sign in with GOV.UK Verify
       url: https://www.tax.service.gov.uk/check-income-tax/start-verify?_ga=2.265317837.398886849.1513589676-373904926.1473694521
       hint_text: You’ll have an account if you’ve already proved your identity with either Barclays, CitizenSafe, Digidentity, Experian, Post Office, Royal Mail or SecureIdentity.
+    - text: Sign in with a digital identity from another European country
+      url: https://www.tax.service.gov.uk/check-income-tax/start-verify?journey_hint=eidas_sign_in
+      hint_text: If you are part of Germany’s Online-Ausweis you can use it here. Identity schemes from other countries will become available later.
```

Comment:
I think you'll want to update the `change_note` below with this latest change.

Reply:
And similar for the other files.

Reply:
I've added new values for the change_note at the bottom of affected files.